dvclive/keras.py

```diff
@@ -14,6 +19,17 @@ def __init__(
         self.save_weights_only = save_weights_only
         self.dvclive = Live(**kwargs)
 
+    def on_train_begin(self, logs=None):
+        if (
+            self.dvclive._resume
+            and self.model_file is not None
+            and os.path.exists(self.model_file)
```

Comment:
Maybe it would be good to also have tests for each of these conditions (for example, model loading isn't called if `dvclive._resume == False`)?